client/src/pages/CreateProduct.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button, Container, Paper, Grid, TextField, InputAdornment, FormControl, InputLabel, OutlinedInput } from "@material-ui/core";
+import '../App.css';
+
+const useStyles = makeStyles((theme) => ({
+  rootHeader: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+  },
+  rootPaper: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    '& > *': {
+      margin: theme.spacing(1),
+      width: theme.spacing(16),
+      height: theme.spacing(16),
+    },
+  },
+  rootForm: {
+    '& > *': {
+      margin: theme.spacing(1),
+    },
+  },
+}));
+
+const typeOfProduct = [
+  {
+    value: 'Cake',
+    label: 'Cake',
+  },
+  {
+    value: 'Cupcake',
+    label: 'Cupcake',
+  },
+  {
+    value: 'Macarons',
+    label: 'Macarons',
+  },
+  {
+    value: 'Cookies',
+    label: 'Cookies',
+  },
+  {
+    value: 'Confections',
+    label: 'Confections',
+  },
+];
+
+export default function CreateProduct() {
+  const classes = useStyles();
+
+  return (
+    <div>
+        <Container>
+            <Grid container spacing={1}>
+              <Grid item xs={6}>
+                <h1 className="margin-top-60">Upload new product</h1>
+
+                <div className={classes.rootPaper + ' margin-top-30'}>
+                  <Paper elevation={3} />
+                  <Paper elevation={3} />
+                  <Paper elevation={3} />
+                </div>
+                <div className={classes.rootPaper}>
+                  <Paper elevation={3} />
+                  <Paper elevation={3} />
+                  <Paper elevation={3} />
+                </div>
+              </Grid>
+              <Grid item xs={6}>
+                <form className={classes.rootForm + " margin-top-140"}>
```

Comment:
I would avoid using a mix of App.css and makeStyles, stick with one. so if you need 140px margin on, i would add it to rootForm, or create another attribute to makeStyles